cfme/scripting/release.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
+import re
+import sys
+from collections import defaultdict
+
+import click
+import github
+import tabulate
+
+from cfme.utils.conf import docker
+
+
+REPO_NAME = "ManageIQ/integration_tests"
+
+HEADERS = ['PR', 'Labels', 'Author', 'Title']
+FULL_HEADERS = HEADERS + ['DESCRIPTION']
+
+VALID_LABELS = [
+    "ansible",
+    "blackify",
+    "blockers-only",
+    "collections-conversion",
+    "customer-case",
+    "doc",
+    "enhancement",
+    "fix-framework",
+    "fix-locator-or-text",
+    "fix-test",
+    "implement-ssui",
+    "infra-related",
+    "issue-bug",
+    "issue-rfe",
+    "LegacyBranch",
+    "manual",
+    "new-test-or-feature",
+    "Nuage",
+    "other",
+    "py3-compat",
+    "rc-regression-fix",
+    "Redfish",
+    "requirements",
+    "RHV",
+    "sprout",
+    "tech-debt",
+    "test-automation",
+    "test-cleanup",
+    "widgetastic-conversion",
+]
```

Comment:
Is there anyway to fetch these via the GH api? I see there is `repo.get_labels()`, this returns a paginated list of GH label objects. Hmm, but you still want a list of `valid_labels`. Could we get away with just a list of `invalid_labels`?

Reply:
We can get them via the API, this is kind of filtering out `RFR`, `1LP`, `WIPTEST` kind of labels.

I think its worth further discussion, okay with it being in a followup (along with additional stats?)

Reply:
Yep, a followup is cool with me